vars/MavenStagingHelper.groovy

```diff
@@ -0,0 +1,111 @@
+def class MavenStagingHelper {
+
+    def steps
+
+    MavenCommand mvnCommand
+
+    String nexusReleaseUrl = 'https://repository.jboss.org/nexus'
+    String nexusReleaseRepositoryId = 'jboss-releases-repository'
+
+    // If not defined, will retrieve default from project artifactId & version
+    String stagingDescription
+
+    // Will be filled once `stageLocalArtifacts` is called or via the `withStagingRepositoryId` method
+    String stagingRepositoryId
+
+    MavenStagingHelper(steps){
+        this(steps, new MavenCommand(steps))
+    }
+
+    MavenStagingHelper(steps, mvnCommand){
+        this.steps = steps
+        this.mvnCommand = mvnCommand
+    }
+
+    // Return a Map of staging properties
+    Map stageLocalArtifacts(String stagingProfileId, String localArtifactsFolder){
```

Comment:
Could you please add a comment explaining how it's supposed to work with this method? If I understand it correctly:
1/ `mvn deploy` (to a local repository using the `altDeploymentRepository` property)
2/ call this method, which pushes the artifacts to the staging repository on the JBoss Nexus
3/ call `promoteStagingRepository`, which promotes the staging repository to a public repo on the JBoss Nexus